### PR TITLE
Task/add commands

### LINKED ISSUE #108: How to send commands to devices?
Hello,
what is the way to send commands or attribute updates to devices?

For example, user sets temp to 24 degrees via UI app, and this information must be propagated to the MQTT-connected actuator.

is there something like `/1234/actuator01/commands` or should we use SUB on `/1234/actuator01/attributes`?

If I understand well `mosquitto_pub -t /1234/sensor01/configuration/commands -m '{ "type": "configuration", "fields": [ "sleepTime" ] }'` is the way to send command IOTA -> Orion, but what is the correct way to send the command IOTA -> physical_device?

### PR DESCRIPTION
**Pull request by dmoranj**
Implements commands for the IoT Agent.

Fixes #108

### CODE REVIEW

File: test/deviceProvisioning/provisionConfigurationCommand1.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "services": [
+    {
+      "resource": "/iot/d",
+      "entity_type": "MQTTCommandDevice",
+      "apikey": "ALTERNATIVE",
+      "protocol": "MQTT_UL"
+    }
+  ]
+}
```
Comment (gtorodelvalle) [inline test/deviceProvisioning/provisionConfigurationCommand1.json]: Final blank line appreciated... :)

File: lib/iotagent-mqtt.js [hunk 2]
```diff
@@ -26,6 +26,7 @@
 var iotAgentLib = require('iotagent-node-lib'),
     mqtt = require('mqtt'),
     dateFormat = require('dateformat'),
+    commandHandler = require('./commandHandlers'),
```
Comment (gtorodelvalle) [inline lib/iotagent-mqtt.js]: It felt awkward to me when I read that the new file was called `commandHandlers.js` versus 'commandHandler.js`. Any reason to use the plural here?

Comment (dmoranj) [inline lib/iotagent-mqtt.js]: Not really. Changed in aa0c46f

**Comment by gtorodelvalle** [comment]
LGTM

**Pull request merged**